src/contexts/providers.context.tsx

```diff
@@ -109,76 +160,50 @@ const ProvidersProvider: FC<PropsWithChildren> = (props) => {
             if (isMetaMaskResourceUnavailableError(error)) {
               return setAccount({
                 status: "failed",
-                error: `Please unlock ${WalletName.METAMASK} to continue`,
+                error: `Please connect to or unlock ${WalletName.METAMASK} to continue`,
```

Comment:
`Please unlock or connect to...` sounds better.

Reply:
Yep, done.